submissions/Oleksandr-Heleta/OOP_exercise/index.js

```diff
@@ -28,7 +28,9 @@ class Human extends Inhabitants {
    }
 
    createString() {
-      return `${this.species}, ${this.name}, ${this.gender}, ${this.legs}, ${this.hands}, ${this.say};`;
+      const superArr = [...super.createString()];
+      superArr.splice(4, 0, this.hands);
```

Comment:
I see what you mean and why changed the `createString` behaviour.
Might work. Although not really scalable.
What if base class developer (imagine you are on a big team here) changes the order of properties?
This will ruin your logic here.
So we either do not rely on base class implementation or we want the base class to be able to provide us with a info about where it is safe to insert extra items at.